packages/oruga-next/src/components/dropdown/Dropdown.vue

```diff
@@ -223,6 +224,22 @@ const props = defineProps({
         type: [String, Array, Function] as PropType<ComponentClass>,
         default: undefined,
     },
+    useRootPositionClass: {
```

Comment:
Unnessaray, just provide poth classes if defined. They are always an empty string if not defined and override is true (with the theme has).